src/constructs/lambda/lambda.ts

```diff
@@ -108,6 +139,21 @@ export class GuLambdaFunction extends Function {
       });
     }
 
+    if (props.config) {
+      // For details, see https://github.com/guardian/ssm-to-env.
+
+      const configPrefix = props.config.ssmPrefix ?? `/${scope.stage}/${scope.stack}/${this.app}`;
+      const configLayer = LayerVersion.fromLayerVersionArn(
+        scope,
+        "config-layer",
+        "arn:aws:lambda:eu-west-1:ACCOUNT-INFO-TODO:layer:ssm-to-env-layer:6" // todo how to include account number here?
+      );
+
+      this.addLayers(configLayer);
```

Comment:
I think it would be useful to understand how this layer will play with other layers that a consumer might add. I don't think I've yet thought that through in the `ssm-to-env` implementation (though it's unlikely to a problem quickly as mutliple layers is probably in our infra is so rare as to not be a concern).

Reply:
I've added a comment to warn users that this will not work if they have their own wrapper script defined. I agree that these cases are likely to be very rare though so not too concerned.